meson_init.sh

```diff
@@ -94,6 +100,26 @@ if [[ "${FLAGS_assert}" == true ]]; then
   fi
 fi
 
+TOCK_LOCAL=false
+if [[ ! -z "${ARG_tock_dir}" ]]; then
+  TOCK_LOCAL=true
+
+  if [[ ! -d "${ARG_tock_dir}" ]]; then
+    echo "Referenced Tock directory ${ARG_tock_dir} doesn't exist. Aborting." >&2
+    exit 1
+  fi
+
+  if [[ ! -f "${TOCK_SYMLINK}" || -L "${TOCK_SYMLINK}" ]]; then
+    echo "Creating symlink to Tock project at ${ARG_tock_dir}."
+    ln -sf "${ARG_tock_dir}" "${TOCK_SYMLINK}"
+  elif [[ -d "${TOCK_SYMLINK}" ]]; then
+    echo "Tock directory $TOCK_SYMLINK exists in place. Leaving as is."
+  else
+    echo "Placing Tock at ${TOCK_SYMLINK} would clobber something. Aborting." >&2
+    exit 1
+  fi
+fi
+
```

Comment:
This probably could be split out in it's own `.sh `file that is called from `meson_init.sh`.

Reply:
Fair point, no reason to add unnecessary clutter to `meson_init.sh`.